guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -14,18 +14,20 @@ include::snip_firewalld.adoc[]
 [options="nowrap"]
 ----
 # firewall-cmd \
---add-port="53/udp" --add-port="53/tcp" \
---add-port="67/udp" \
---add-port="69/udp" \
---add-port="80/tcp" --add-port="443/tcp" \
+--add-service=dns \
+--add-service=dhcp \
+--add-service=tftp \
+--add-service=http \
+--add-service=https \
 ifdef::katello,satellite,orcharhino[]
 --add-port="5647/tcp" \
---add-port="8000/tcp" --add-port="9090/tcp" \
+--add-port="8000/tcp" \
```

Comment:
Same here: port 8000 is the HTTP port of foreman-proxy, but that's not enabled by default in katello and satellite (not sure about orcharhino).

Reply:
Enabled via `--foreman-proxy-http true` (and on by default for the content proxies/capsules).